PHP/hw1.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+$sum = 0; 
+for ($i=0; $i<=51; $i++) if($i%2!=0) $sum += $i;
```

Comment:
Please try to follow coding standard at https://www.php-fig.org/psr/psr-2/ 
Let take a look on `if-else` and `for`.
Following coding standard helps your source code more clear.